app/src/main/java/com/example/shorebuddy/views/LakeSelectFragment.java

```diff
@@ -64,7 +66,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             @Override
             public boolean onQueryTextSubmit(String query) {
                 mainViewModel.setCurrentSelectedLakeFromFilteredPosition(0);
-                NavDirections action = LakeSelectFragmentDirections.actionLakeSelectFragmentToMainFragment();
+                //NavDirections action = LakeSelectFragmentDirections.actionLakeSelectFragmentToMainFragment();
+                NavDirections action = LakeFragmentDirections.actionLakeFragmentToLakeSelectFragment();
```

Comment:
This should go to the home fragment

Reply:
That's weird, I have that correction on mine, but it did not push to github